workflows/pkg/apis/workflows/v1alpha1/workflow_types.go

```diff
@@ -37,12 +40,33 @@ type Workflow struct {
 
 // WorkflowSpec describes the desired state of the Workflow
 type WorkflowSpec struct {
+	// TODO: Repositories
+	Secrets []Secret `json:"secrets,omitempty"`
+
+	// Params define the default values for params in the Pipeline that can be
+	// overridden in a WorkflowRun or (in the future) from an incoming event.
+	Params []pipelinev1beta1.ParamSpec `json:"params,omitempty"`
+
+	// Pipeline is a reference to a Pipeline. Currently only an inline
+	// pipelineSpec is supported
+	Pipeline PipelineRef `json:"pipeline,omitempty"`
```

Comment:
Are we going to support only single Pipeline?

Reply:
At the moment that is the plan - one workflow maps to a single pipeline (but with potentially multiple triggers). We can explore how we can do this with multiple pipelines later.